sirepo/package_data/template/srw/rsoptExport.sh.jinja

```diff
@@ -1,11 +1,27 @@
 #!/bin/bash
-echo "NOTE: remove files from {{ runDir }} first"
+if [ -d "{{ runDir }}" ] && [ -n "$(ls -A {{ runDir }})" ]; then
+   echo "Cleaning run directory {{ runDir }}..."
+   rm {{ runDir }}/{{ pyFileName }}
+   rm {{ runDir }}/run_parallel_python.py
+fi
+echo "Generating data..."
```

Comment:
Quotes unnecessary. I would say running rsopt or better yet tell them the command that's being run.

Reply:
5421ff1